apps/ltn/src/edit/mod.rs

```diff
@@ -393,11 +393,15 @@ impl ChangeFilterType {
                 ]),
                 Widget::vertical_separator(ctx),
                 Widget::col(vec![
-                    Image::from_path(app.session.filter_type.svg_path())
-                        .untinted()
-                        .dims(100.0)
-                        .into_widget(ctx)
-                        .centered_horiz(),
+                    GeomBatch::from(vec![
+                        (match app.session.filter_type {
+                            FilterType::WalkCycleOnly => Texture(1),
+                            FilterType::NoEntry => Texture(2),
+                            FilterType::BusGate => Texture(3),
```

Comment:
At first I made the spritesheet 1 row and 3 columns, but Texture(3) kept just using the 2nd image. I moved the 3rd image to the 2nd row and then it worked. I thought they would be equivalent, but I don't understand the array texture math, so oh well -- it works regardless